challenges/classes.js

```diff
@@ -1,7 +1,27 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class CuboidMaker {
+  constructor(length, width, height) {
+    this.length = length;
+    this.width = width;
+    this.height = height;
+  }
+  volume() { return this.length * this.width * this.height; }
+  surfaceArea() { return (2 * (this.length * this.width + this.width * this.height + this.height * this.length))}
+}
+
 
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+console.log(cuboid.volume()); // 100
```

Comment:
You are calling `cuboid` but it was never defined. Fix it with `const cuboid = new CuboidMaker(4, 5, 5)`.